ci/pipeline.yml

```diff
@@ -27,6 +27,14 @@ resources:
     username: ((github-username))
     password: ((github-password))
     branch: ((branch))
+- name: git-repo-ci-docker
+  type: git
+  icon: github-circle
+  source:
+    uri: ((github-repo))
+    username: ((github-username))
+    password: ((github-password))
+    branch: ci-docker-((branch))
```

Comment:
Is this working if the branch doesn't exist?

Reply:
No, we need to create a long-running branch beforehand. I don't know if there's a way to create branches on the fly in concourse.